docs/interfaces/json-api.md

```
@@ -138,55 +135,48 @@ on | bool | On/Off state of the light. You can also use `"t"` instead of `true`
+pl | -1 to 250 | ID of currently set playlist. _(read-only)_
 pdel | 1 to 250 | Preset ID to delete. Not included in state response.
 nl.on | bool | Nightlight currently active
 nl.dur | 1 to 255 | Duration of nightlight in minutes
-~~nl.fade~~ | ~~bool~~ | ~~If `true`, the light will gradually dim over the course of the nightlight duration. If `false`, it will instantly turn to the target brightness once the duration has elapsed.~~ _Removed in 0.13.0_ (use mode instead)
-nl.mode | 0 to 3 | Nightlight mode (0: instant, 1: fade, 2: color fade, 3: sunrise) (available since 0.10.2)
+nl.mode | 0 to 3 | Nightlight mode (0: instant, 1: fade, 2: color fade, 3: sunrise)
 nl.tbri | 0 to 255 | Target brightness of nightlight feature
 nl.rem | -1 to 15300 | Remaining nightlight duration in seconds, -1 if not active. Only in state response, can not be set.
 udpn.send | bool | Send WLED broadcast (UDP sync) packet on state change
-udpn.recv | bool | Receive broadcast packets
+udpn.recv | bool | read-only: Broadcast packets receiver enable status. To set receive groups, clients must send `udpn.rgrp`.
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_only; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)